### ISSUE TITLE
how to put a border on the unselected tabs of Pivot links

### ISSUE THREAD

**Issue by Ofer-Gal** [microsoft/fluentui#21819]
how to put a border on the unselected tabs of Pivot links?
Thanks

**Comment by tomi-msft** [comment]
Hi @Ofer-Gal, thanks for submitting the issue! Can you provide more details on the problem?

**Comment by Ofer-Gal** [comment]
I wanted to give the unselected tabs of Pivot a border so they look like real tabs. I figured out it is actually styling a button so I created:
`const headerProps: IButtonProps = { 
    styles: {
      flexContainer:{'border': '1px solid '},
    }
  };`

but flexContainer selector does not cover the whole width of the tab.

![image](https://user-images.githubusercontent.com/5054931/155864317-f3ad1b83-bcd8-4945-9105-3d7e15eab923.png)
root does not show border at all. 
there are too many selectors in IButtonStyles to test :-) Anyone knows which it is?
I over write .ms-Pivot-link class but that changes other none related parts of the page.
